schemas/common_definitions.json

```diff
@@ -193,6 +201,11 @@
           "required": [
             "answer_count"
```

Comment:
Pretty sure this is wrong?. This is valid if `oneOf` them is provided but we need more than 1.